ReSharper.FSharp/test/data/parsing/signatures/Type member - XmlDoc 03 - Constructor.fsi

```diff
@@ -0,0 +1,6 @@
+module Module
+
+type T =
+
+    /// C
+    new: () -> T
```

Comment:
Is it correct syntax at all?

Reply:
Oh, thanks, there must be a `unit` instead of `()`